content/brazilian-portuguese/jsappybird/index_1.md

```diff
@@ -0,0 +1,264 @@
+---
+title: "Javascript: JSappy Bird"
+description: "Aprenda sobre variáveis, funções e condicionais enquanto construímos um jogo parecido com o Flappy Bird."
+date: 2023-09-11T13:24:17-07:00
+difficulty: "Iniciante/Intermediário"
+draft: true
+hidden: true
+---
+
+![alt text](resources/_gen/images/flappy.png "JSappyBird")
+
+1. **Baixe os arquivos necessários.** <link_aqui>
+
+O modelo contém os seguintes arquivos:
+
+- `phaser.min.js`, a estrutura framework v2.4.3.
+- `index.html`, onde o jogo será exibido.
+- `main.js`, o arquivo onde vamos escrever nosso código.
+- `assets/`, uma pasta com 2 imagens.
+- Atividate `1.js`
```

Comment:
I believe this should be translated as "Atividade". Same for the lines 21 and 22.